ui/component/embedTextArea/view.jsx

```diff
@@ -21,7 +21,7 @@ export default function EmbedTextArea(props: Props) {
   const { claim_id: claimId, name } = claim;
   const input = useRef();
 
-  const streamUrl = generateEmbedUrl(name, claimId, includeStartTime, startTime, referralCode);
+  const streamUrl = generateEmbedUrlEncoded(name, claimId, includeStartTime && startTime, referralCode);
```

Comment:
99% of the time, the $ should work here and looks nicer. Let's keep the original function?